rma_product_exchange/wizards/rma_make_picking.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2024 Akretion
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+
+from odoo import api, models
+
+
+class RmaMakePicking(models.TransientModel):
+    _inherit = "rma_make_picking.wizard"
+
+    @api.returns("rma.order.line")
+    def _prepare_item(self, line):
+        values = super()._prepare_item(line)
+        if (
+            self.env.context.get("picking_type") == "outgoing"
+            and line.product_exchange
+            and line.new_product_id
+        ):
+            values["product_id"] = line.new_product_id.id
+        return values
+
+    @api.model
+    def _create_procurement(self, item, picking_type):
```

Comment:
This whole override is useless for now, because `rma_item` context key is not used anywhere anymore, right ?
In if this the case, please, remove it!